src/test/resources/go/complex-user/dao/dao.go.snippet

```diff
@@ -0,0 +1,172 @@
+package dao
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/squat/and/dab/complexuser/util"
+	"github.com/google/uuid"
+
+	// pq acts as the driver for SQL requests
+	_ "github.com/lib/pq"
+)
+
+// Datastore provides the interface adopted by the DAO, allowing for mocking
+type Datastore interface {
+	CreateComplexuser(input CreateComplexuserInput) (*Complexuser, error)
```

Comment:
Don't think this is you, probably @smithwjv - could we make this `ComplexUser`?

Reply:
Should be handled if it comes in as `complexUser`

Reply:
It capitalises whatever comes in for the service name